packages/@react-spectrum/listbox/test/ListBox.test.js

```diff
@@ -763,5 +763,32 @@ describe('ListBox', function () {
 
       expect(onLoadMore).toHaveBeenCalledTimes(1);
     });
+
+    it('should fire onLoadMore if there aren\'t enough items to fill the ListBox ', function () {
```

Comment:
Bit iffy on these tests (listbox and table) now that I can't separate the loadMore calls that come from the onVisibleRectChange from the ones triggered by useLayoutEffect. Trying to think of a good way to test that (as well as to simulate the isLoading/animating/etc logic so that only one loadMore is triggered from useLayoutEffect)